src/locales/en.json

```diff
@@ -68,6 +69,7 @@
     "address": "",
     "cozy": "",
     "company": "",
+    "jobTitle": "",
```

Comment:
why 2 additions ?

Reply:
Il y en a un pour le label, et un pour le placeholder

Reply:
Nit: AMA it would be better to keep both labels and placeholder at the same place.

```
fields: {
  jobTitle: {
    label: "Job title"
    placeholder: ""
  }
}
```

It's easier to see values that are related. This way, if we remove the jobTitle.label, we can see that we should also remove the jobTitle.placeholder. It also makes it easier to move code around.

Reply:
If you have one for the label and one for the placeholder, it seems that you missed to add one for fr & es.json

Reply:
@ptbrowne  Ok je déplace.

@Crash-- pour les placeholder manquant, comme ils servent à forcer un champ vide, je suppose que c'était inutile de les mettre dans les `fr` et `es`. Le comportement doit être : s'il n'y a rien, il va chercher dans `en`, et comme c'est pour forcer un champ vide, ça fonctionne aussi. Mais ok je les rajoute pour que les traductions soient autonome.

Reply:
@Crash-- @ptbrowne j'ai finalement supprimé les placeholder qui ne servent plus à rien avec les nouveaux styles d'input. Ça me paraît plus sain de supprimer des choses inutiles quitte à les remettre par la suite si jamais on veut finalement s'en servir, que de garder une fonctionnalité qui ne sera peut-être jamais utilisée. Qu'en dites-vous ?